packages/manager/apps/web/client/app/domain/domain.service.js

```diff
@@ -1392,6 +1392,17 @@ angular.module('services').service(
 
     //----------------------------------------------------
 
+    /**
+     * Get attached hosting to domain
+     * @param {string} serviceName
+     */
+    getAssociatedHosting(domain) {
+      return this.OvhHttp.get('/hosting/web/attachedDomain', Object.assign({
```

Comment:
Could we use an `ovh-api-services` method here ?

Reply:
https://github.com/ovh-ux/ovh-api-services/pull/259